src/main.py

```diff
@@ -263,6 +263,9 @@ def __init__(self, aconfig):
 
 		self._aconfig = aconfig
 
+		if not self._aconfig['refresh_token']:
```

Comment:
Yeah, that's what I meant. I thought I'd check before commiting it - too late :smile: